DDG4/plugins/Geant4.10.PhysicsConstructors.h

```diff
@@ -2,7 +2,7 @@
 #define DDG4_PLUGINS_GEANT4_10_PHYSICSCONSTRUCTORS_H
 
 //==========================================================================
-//  AIDA Detector description implementation 
+//  AIDA Detector description implementation
```

Comment:
Was this file really supposed to be included in this MR ? I do not see the relationship